### PR TITLE
Add SECURITY.md file

### LINKED ISSUE #35: Add security policy
See https://help.github.com/en/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository

### PR DESCRIPTION
**Pull request by whymarrh**
Closes #35

This PR adds a `SECURITY.md` file, loosely based on:

- [lodash's `SECURITY.md` file](https://github.com/lodash/lodash/blob/master/SECURITY.md)
- [Agoric's `SECURITY.md` file for SES-shim](https://github.com/Agoric/SES-shim/blob/master/SECURITY.md)

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Reporting a Security Issue
+
+We consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present.
+
+If you discover a security vulnerability, please use one of the following means of communications to report it to us:
+
+- Report the security issue to the Node.js Security Working Group through the [HackerOne program](https://hackerone.com/nodejs-ecosystem) for ecosystem modules on npm, or to [Snyk Security Team](https://snyk.io/vulnerability-disclosure). They will help triage the security issue and work with all involved parties to remediate and release a fix.
+
+- Report the security issue to the MetaMask team directly at [security@metamask.io](mailto:security@metamask.io).
```
Comment (whymarrh) [inline SECURITY.md]: I don't know that this is a real email address yet

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+We kindly ask you to refrain from malicious acts that put our users, the project, or any of the project's team members at risk.
+
+## Reporting a Security Issue
+
+We consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present.
+
+If you discover a security vulnerability, please use one of the following means of communications to report it to us:
+
+- Report the security issue to the Node.js Security Working Group through the [HackerOne program](https://hackerone.com/nodejs-ecosystem) for ecosystem modules on npm, or to [Snyk Security Team](https://snyk.io/vulnerability-disclosure). They will help triage the security issue and work with all involved parties to remediate and release a fix.
```
Comment (Gudahtt) [inline SECURITY.md]: I'm guessing this step is more relevant for Lodash than it is for us. I'm not sure they'd want our product-specific security advisories.

Comment (whymarrh) [reply SECURITY.md]: This is for libraries that could be used by anyone. This definitely wouldn't be useful for the extension or mobile apps but for libs this seems to make sense to me

Comment (Gudahtt) [reply SECURITY.md]: I suppose so. It's just that a lot of the stuff we'd be likely to encounter is kinda... in the weeds? But I guess it depends on the issue.

**Comment by whymarrh** [comment]
Replaced with an org-wide file: https://github.com/MetaMask/.github/pull/1

**Pull request closed without merging**